### PR TITLE
Consolidated and refactored the database locking methods (0.9.54)

### PR DESCRIPTION
**Pull request by Ceiu**
- Several methods used for applying database/entity locks have been
  generalized and refactored; extraneous methods have been removed

### CODE REVIEW

File: server/src/main/java/org/candlepin/controller/CandlepinPoolManager.java [hunk 1]
```diff
@@ -1306,20 +1302,19 @@ public void revokeEntitlements(List<Entitlement> entsToRevoke) {
             poolIdsToLock.add(ent.getPool().getId());
         }
 
-        poolCurator.lockAndLoadBatchById(poolIdsToLock);
+        poolCurator.lockAndLoadByIds(poolIdsToLock);
 
-        log.info("Batch revoking entitlements: " + entsToRevoke.size());
+        log.info("Batch revoking entitlements: {}", entsToRevoke.size());
```
Comment (awood) [inline server/src/main/java/org/candlepin/controller/CandlepinPoolManager.java]: 💵 💎 💰 💴

**Comment by kahowell** [comment]
test this please

**Comment by Ceiu** [comment]
The qpid spec test will fail in this branch, as qpid previous existed in the gutterball directory rather than the candlepin directory where the current test infrastructure is expecting it.

**Comment by Ceiu** [comment]
@candlepin-pull-request-bot retest this please

**Pull request merged**